
# HG changeset patch
# User David Bienvenu <bienvenu@nventure.com>
# Date 1236181428 28800
# Node ID 513295ae0230e3d0baa73b1ee98627d4b2f97d24
# Parent f290d90a853210f29f7b9306abf657dd81f0d6b5
fix crash on startup when imap server sends list response with trailing delimiter (not /), r/sr=standard8, 480870

--- a/mailnews/imap/src/nsImapIncomingServer.cpp	Wed Mar 04 10:59:11 2009 +0000
+++ b/mailnews/imap/src/nsImapIncomingServer.cpp	Wed Mar 04 07:43:48 2009 -0800
@@ -1109,7 +1109,7 @@ NS_IMETHODIMP nsImapIncomingServer::Poss
     return rv;
 
   nsCAutoString dupFolderPath(folderPath);
-  if (dupFolderPath.Last() == hierarchyDelimiter)
+  if (dupFolderPath.Last() == '/')
   {
     dupFolderPath.SetLength(dupFolderPath.Length()-1);
     // *** this is what we did in 4.x in order to list uw folder only

